_data/errors.yml

```diff
@@ -118,4 +118,78 @@ oidc:
+        ##### What to do:
+        - Attempt another authentication attempt without pauses.
+    - title: Invalid audience claim
+      id: invalid-aud-claim
+      accordion-id: invalid_aud_claim
+      content: |
+        ##### Why it's happening
+        The `aud` value in the `client_assertion` JWT is not correct.
+        ##### What to do:
+        - Check the `aud` key in the `client_assertion` JWT that is being sent in your `token` request.
+        Integration environment value: https://idp.int.identitysandbox.gov/api/openid_connect/token
+        Production environment value: https://secure.login.gov/api/openid_connect/token
+
+        More information about the client assertion values can be found here: [https://developers.login.gov/oidc/token/#client_assertion](https://developers.login.gov/oidc/token/#client_assertion){:target="_blank"}.
+    - title: Client must authenticate via PKCE or private_key_jwt, missing either code_challenge or client_assertion
+      id: pkce-or-private-key
+      accordion-id: pkce_or_private_key
+      content: |
+        ##### Why it's happening
+        OpenID Connect authentications must use either a PKCE or a private_key_jwt authentication menthod. Both methods require specific values to be sent in `token` endpoint request.
+        ##### What to do:
+        If your integration is PKCE:
+        - Ensure you are sending the `code_verifier` attribute as part of your token request. The value of this attribute is the original value of the `code_challenge` parameter sent in your original authentication request.
+        If your integration is private_key_jwt:
+        - Ensure you are sending `client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer` as part of your token request.
+
+        More information about token request values can be found here: [https://developers.login.gov/oidc/token/#request-parameters](https://developers.login.gov/oidc/token/#request-parameters){:target="_blank"}.
+    - title: Code is invalid because doesn’t match any user. Please see our documentation at https://developers.login.gov/oidc/#token
+      id: invalid-code
+      accordion-id: invalid_code
+      content: |
+        ##### Why it's happening
+        The `code` value being passed in the token request does not match any users. This typically happens when the code value is being reused in multiple token requests, or because the `code` value is not being passed in the token request.
+        ##### What to do:
+        - Ensure you are passing a `code` parameter in your token request.
+        - Ensure there is no possibility of the `code` value in the token request being sent multiple times.
+
+        Reasons can include:
```

Comment:
This seems out of place here though I am not certain. Wanted to flag just in case.

Reply:
i wanted to give a couple hints to the developer since this is hard to debug!